requirements/edx/github.txt

```diff
@@ -55,7 +55,7 @@ git+https://github.com/edx/edx-lint.git@ed8c8d2a0267d4d42f43642d193e25f8bd575d9b
 -e git+https://github.com/edx/edx-reverification-block.git@a286e89c73e1b788e35ac5b08a54b71a9fa63cfd#egg=edx-reverification-block
 git+https://github.com/edx/ecommerce-api-client.git@1.1.0#egg=ecommerce-api-client==1.1.0
 -e git+https://github.com/edx/edx-user-state-client.git@64a8b603f42669bb7fdca03d364d4e8d3d6ad67d#egg=edx-user-state-client
--e git+https://github.com/edx/edx-organizations.git@release-2015-07-17#egg=edx-organizations
+-e git+https://github.com/edx/edx-organizations.git@4b3cb2da2a94ad292321d9602e5a5992bffb1cc8#egg=edx-organizations
 
```

Comment:
we will update this hash after merging of https://github.com/edx/edx-organizations/pull/3